src/main/java/org/mate/bookstorespringboot/controller/dto/cartitem/CartItemRequestDto.java

```diff
@@ -0,0 +1,8 @@
+package org.mate.bookstorespringboot.controller.dto.cartitem;
+
+import jakarta.annotation.Nonnull;
+import jakarta.validation.constraints.Min;
+
+public record CartItemRequestDto(@Min(1) int quantity,
+                                 @Nonnull Long bookId) {
```

Comment:
Better use annotation from validators package.

Reply:
Sorry, don't quite understand what is the validators package?